docs/funding/index.md

```diff
@@ -20,4 +20,8 @@ This section lists past and ongoing funding for Jupyter Accessibility.
   - Cycle 4 - awarded to [Quansight Labs](https://labs.quansight.org/)
   - 2021-2023
   - [Submitted proposal](https://github.com/jupyter/accessibility/blob/1e048c086782cb10848292c4befbe09019853f96/docs/funding/Inclusive_and_Accessible_Scientific_Computing_in_Jupyter_Ecosystem_SUBMITTED_PROPOSAL.pdf) and [corresponding execution roadmap](czi-grant-roadmap.md)
+* - [Astronomy Notebooks For All - Space Telescope Science Institute (STScI)](https://github.com/Iota-School/notebooks-for-all#astronomy-notebooks-for-all)
+  - Internal grant at STScI
+  - 2022-2023
+  - [Submitted proposal](https://github.com/Iota-School/notebooks-for-all/blob/main/resources/proposal-astronomy-notebooks-for-all.md) (minus personal information)
```

Comment:
Permalink in case anyone ever decides to move the files around in that repo:

```suggestion
  - [Submitted proposal](https://github.com/Iota-School/notebooks-for-all/blob/0c06d4fb037544fa96d3f90a1bb6efe7570a59a8/resources/proposal-astronomy-notebooks-for-all.md) (minus personal information)
```